src/System.Net.Security/src/System/Net/Windows/CertModule.cs

```diff
@@ -154,6 +158,69 @@ internal override string[] GetRequestCertificateAuthorities(SafeDeleteContext se
             return issuers;
         }
 
+        //
```

Comment:
It seems to be that way; flipping back and forth between the add here and the delete in the other file, the only difference is here is override, there was static.